### ISSUE TITLE
[FEATURE REQUEST] add an option for generating all the variations from 1 seed from all the classifier or steps or both 

### ISSUE THREAD

**Issue by search620** [hlky/stable-diffusion-webui#200]
add an option for generating all the variations from 1 seed from all the classifier or steps or both 

It would be really great to have the ability to select a seed and then manually choose how much variation to generate from it by classifier or steps or both in a row. 

At the current status if I want to do it then I have to do it by hand step by step manually (time-consuming)
The reason I'm interested in a feature like this is that sometimes there are good seeds and I noticed that If I played a little bit with the sliders of the steps or the classifier then the results can be a great outcome.

What do you think about this type of feature?

**Comment by hlky** [comment]
Yes I thought about this, I would probably make it a standalone script to avoid complicating the main ui and keep it out the way of people who don't need it

I have noticed the same as you different cfg and steps on the same seed can be different results.

My thoughts were if you found an image, had the seed and prompt, but not the cfg, you could 'bruteforce' by trying all the possible cfg values, this might be useful if the image file is lost, or an image you find you like that has since been watermarked, or something like that. Same thing basically but different usage

Btw I am thinking about moving feature requests to discussions, each feature request can have a discussion and then there can be a main discussion for new features with a list.
In fact, I will move this to discussions now.
